packages/react-microfrontend/src/controller/index.tsx

```diff
@@ -6,7 +6,24 @@ import CreateLib from '../state/createLib';
 const shared = new Shared('__core__');
 const microfrontendFolderName = 'microfrontends';
 
+declare global {
+  interface Event {
+    data?: any
+  }
+}
+
 class Controller {
+  __onMicrofrontendsInfosLoaded(microfrontends: any) {
```

Comment:
this was originally private with "__" prefix. If we are going to use typescript, maybe we should set this as private with typescript syntax